src/themes/utils.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { px, py } from './utils'
+
+describe( 'Styling Utilities', () => {
+  it( 'should have paddingBottom and paddingTop set to auto', () => {
+    expect( py() ).toStrictEqual( { paddingTop: 'auto', paddingBottom: 'auto' } )
```

Comment:
Why `strictEqual` over `equal`? Had to look up what `.toStrictEqual` does, and don't seem to see the benefit.